src/View/TwigView.php

```diff
@@ -118,17 +124,21 @@ public function initialize(): void
      */
     public function getTwig(): Environment
     {
-        return $this->twig;
+        if (static::$twig === null) {
+            throw new RuntimeException('Twig Environment instance not created.');
+        }
+
+        return static::$twig;
     }
 
     /**
      * Gets Twig Profile if profiler enabled.
      *
-     * @return \Twig\Profiler\Profile
+     * @return \Twig\Profiler\Profile|null
      */
-    public function getProfile(): Profile
+    public function getProfile(): ?Profile
```

Comment:
Do you see a need for this method an maintaining reference to the profile instance? Once it's passed to the `ProfileExtension` constructor I don't see it used anywhere. Is the app/test code supposed to use it somehow?

Reply:
https://twig.symfony.com/doc/3.x/api.html#profiler-extension

I saw that the Profile instance can be used in other code that the user might add to their override of initializeProfiler(). I wanted to give them a way to access it. I don't have any code using it right now.

Reply:
Okay, I found following code in symfony docs, so it's reference is required.

```php
$profile = new Twig_Profiler_Profile();
$twig->addExtension(new Twig_Extension_Profiler($profile));

// render templates ...

$dumper = new Twig_Profiler_Dumper_Text();
echo $dumper->dump($profile);
```